packages/kit/package.json

```diff
@@ -12,37 +12,33 @@
 	"dependencies": {
 		"@sveltejs/vite-plugin-svelte": "^1.0.1",
 		"chokidar": "^3.5.3",
+		"cookie": "^0.5.0",
+		"devalue": "^2.0.1",
+		"kleur": "^4.1.4",
+		"magic-string": "^0.26.2",
+		"mime": "^3.0.0",
+		"node-fetch": "^3.2.4",
 		"sade": "^1.8.1",
-		"tiny-glob": "^0.2.9"
+		"set-cookie-parser": "^2.4.8",
+		"sirv": "^2.0.2",
+		"tiny-glob": "^0.2.9",
+		"typescript": "^4.7.4",
```

Comment:
oof

Reply:
sorry, should have been more wordy :see_no_evil: 
is it possible to not depend on typescript here? That's not small and even if users mostly are going to have it around during dev anyways, outright depending on it would lead to `npm i --production` pulling it too, which i think is not good / too heavy

Reply:
this is the plan, yeah — it's only necessary for generated types, and if you don't have typescript in your app then there's no point in generating those types. so ideally we'll depend on the app's copy of typescript. i was just too lazy to do that